projects/openjdk.org/package.yml

```diff
@@ -81,16 +92,18 @@ build:
       JDK_DIR: build/*/images/jdk-bundle/jdk-{{version.marketing}}.{{version.patch}}.jdk/Contents/Home/*
       ARGS:
         - --enable-dtrace
+        # - --with-sysroot=#{MacOS.sdk_path} # from homebrew's formulae
     linux:
       STRIP: 1
       BOOT_JDK_DIR: $SRCROOT/boot-jdk
       JDK_DIR: build/*/images/jdk/*
       ARGS:
+        - --with-x={{deps.x.org/x11.prefix}} # from homebrew's formulae
         - --with-cups={{deps.openprinting.github.io/cups.prefix}}
         - --with-fontconfig={{deps.freedesktop.org/fontconfig.prefix}}
         - --with-freetype=system
         - --with-stdc++lib=dynamic
-        - --with-toolchain-type=clang
+        - --with-toolchain-type=clang # not in brew's formulae
```

Comment:
what do you think about this one? not in homebrew's formulae

Reply:
we try to use clang throughout; if it doesn't require that flag, then it should be ok to omit.